acquisitions/src/main/resources/thunderjet/mod-invoice/features/invoice-fiscal-years.feature

```diff
@@ -0,0 +1,141 @@
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+    * def currentYear = callonce getCurrentYear
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
+    * def createInvoice = read('classpath:thunderjet/mod-invoice/reusable/create-invoice.feature')
+    * def createInvoiceLine = read('classpath:thunderjet/mod-invoice/reusable/create-invoice-line.feature')
+
+    * def pastFiscalYearId1 = callonce uuid1
+    * def pastFiscalYearId2 = callonce uuid2
+    * def pastFiscalYearId3 = callonce uuid3
+    * def presentFiscalYearId = callonce uuid4
+    * def futureFiscalYearId = callonce uuid5
+    * def ledgerId = callonce uuid6
+    * def fundId1 = callonce uuid7
+    * def fundId2 = callonce uuid8
+    * def fundId3 = callonce uuid9
+    * def fundId4 = callonce uuid10
+    * def fundId5 = callonce uuid11
+    * def budgetId1 = callonce uuid12
+    * def budgetId2 = callonce uuid13
+    * def budgetId3 = callonce uuid14
+    * def budgetId4 = callonce uuid15
+    * def budgetId5 = callonce uuid16
+    * def budgetId6 = callonce uuid17
+    * def budgetId7 = callonce uuid18
+    * def budgetId8 = callonce uuid19
+    * def budgetId9 = callonce uuid20
+    * def budgetId10 = callonce uuid21
+    * def budgetId11 = callonce uuid22
+    * def budgetId12 = callonce uuid23
+    * def budgetId13 = callonce uuid24
+    * def budgetId14 = callonce uuid25
+    * def invoiceId1 = callonce uuid26
+    * def invoiceId2 = callonce uuid27
+    * def invoiceId3 = callonce uuid28
+    * def invoiceLineId1 = callonce uuid29
+    * def invoiceLineId2 = callonce uuid30
+    * def invoiceLineId3 = callonce uuid31
+    * def invoiceLineId4 = callonce uuid32
+    * def invoiceLineId5 = callonce uuid33
+
+
+  Scenario: Create finances
+    * configure headers = headersAdmin
+    # 3 FY in the past (including one with a different series), 1 in present, 1 in the future
+    * def v = call createFiscalYear { id: #(pastFiscalYearId1), code: 'TESTFY2020', periodStart: '2020-01-01T00:00:00Z', periodEnd: '2020-12-30T23:59:59Z', series: 'TESTFY' }
+    * def v = call createFiscalYear { id: #(pastFiscalYearId2), code: 'TESTFY2021', periodStart: '2021-01-01T00:00:00Z', periodEnd: '2021-12-30T23:59:59Z', series: 'TESTFY' }
+    * def v = call createFiscalYear { id: #(pastFiscalYearId3), code: 'FY2022', periodStart: '2022-01-01T00:00:00Z', periodEnd: '2022-12-30T23:59:59Z', series: 'FY' }
+    * def currentStart1 = currentYear + '-01-01T00:00:00Z'
+    * def currentEnd1 = currentYear + '-12-30T23:59:59Z'
```

Comment:
Will this test work in the next year?

Reply:
It should keep working next year, so this fiscal year is always current. Although I should probably use the year in the code for clarity. I will change that.